tests/sync/test_tokenize.py

```diff
@@ -8,6 +9,17 @@
 
 
 class TestTokenize(unittest.TestCase):
+    @pytest.mark.skipif(True, reason="until this is available globally in the API")
```

Comment:
I believe the sync tests don't use pytest. Instead you can skip the test using the unittest decorators: https://docs.python.org/3/library/unittest.html#skipping-tests-and-expected-failures

Reply:
@harry-cohere we still use pytest as our runner, so the decorators will take effect normally:
https://github.com/cohere-ai/cohere-python/blob/main/.github/workflows/test.yaml#L29-L32